src/workerd/io/compatibility-date-test.c++

```diff
@@ -31,6 +31,7 @@ KJ_TEST("compatibility date parsing") {
   expectParseTo("2000-01-01", "2000-01-01");
   expectParseTo("2999-12-31", "2999-12-31");
   expectParseTo("2024-02-29", "2024-02-29");
+  expectParseTo("2112-04-01", "2112-04-01");
```

Comment:
Expecting a  comment on this, wanted to add an Easter egg

https://www.youtube.com/watch?v=AZm1_jtY1SQ